prompt.go

```diff
@@ -16,7 +16,16 @@ func prompt(sources map[Resource]bool, destinations map[Resource]bool) (Resource
 		Selected: "{{ .Address }}",
 	}
 	srcs := toSlice(sources)
-	prompt := promptui.Select{Label: "Select Source", Items: append(srcs, Resource{"Finished", "no more resources to move"}), Templates: srcTempl}
+	srcSearcher := func(input string, index int) bool {
+		if index >= len(srcs) {
```

Comment:
If I understand https://github.com/manifoldco/promptui/blob/master/list/list.go#L82 correctly, this check might be a bit defensive - what do you think?
If we need it, should we also add a corresponding check in `destSearcher` below?

Reply:
We only need it because of the additional `Resource{"Finished", "no more resources to move"}` item that's added to the end of the list.

Reply:
I see, thanks for the explanation!